spec/integration/support/kiwi_export_examples.rb

```diff
@@ -15,27 +15,38 @@
 # To contact SUSE about this file by physical or electronic mail,
 # you may find current contact information at www.suse.com
 
-shared_examples "kiwi export" do
+shared_examples "kiwi export" do |opts|
+  let(:options) { opts || {
+      machinery_dir: "/home/vagrant/.machinery",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.